docs/source/assemblyguide.rst

```diff
@@ -0,0 +1,339 @@
+
+.. note:: Verletzungsgefahr: Mit der Klinge/dem Bohrer immer nach außen, d.h. vom Körper weg, schneiden oder Bohren!
+
+Dach und Zwischenboden
+----------------------
+
+.. image:: images/build_14_cap.jpg
+   :align: right
+   :width: 20%
+   :alt: Deckel
+
+Für das Dach kann man einen Rohrabschlussstopfen oben vorsichtig über das Rohr schieben, s. Bild rechts.
+
+.. image:: images/build_14_bottle_bottom.jpg
+   :width: 30%
+   :align: left
+   :alt: Zwischenboden
+
+Als Zwischenboden verwenden wir eine Kruke, also eine Plastikdose, die in Apotheken zum Abfüllen von Salben verwendet wird.
+Der rote Deckel kann als Material für eine kabeldurchführungsverkleinernde Zwischenbodenauflage dienen.
+
+Für den Zwischenboden (s. Bild links) wird lediglich das Gewinde vorsichtig mit der Säge abgesägt. Der „Kragen“ muss bestehen bleiben, da er als Anschlag dient.
+
+In den Zwischenboden wird in der Mitte ein Loch zur Kabeldurchführung mit 10 mm gebohrt. Der Stecker des Kabels muss durch das Loch passen.
+
+.. Auskommentiert, da die Variante mit Deckel viel einfacher ist
+  Bei Verwendung einer Kruke als Dach wird der „Kragen“ unter dem Gewinde mit abgesägt, so dass die Dachkruke oben genau in das Rohr hineinpasst:
+
+  .. image:: images/build_14_bottle_top.jpg
+     :width: 30%
+     :align: left
+     :alt: Deckel zum Einkleben
+
+  Das obere Viertel der Dachkruke wird seitlich mit Silikonklebstoff benetzt und in das obere Teil des Elektronikgehäuses geschoben. Die Fuge wird weiterhin mit Silikonklebstoff abgedichtet. 
+
+  Auch der obere Rand der Dachkruke wird mit Klebstoff benetzt (s. Skizze). Um ein Hereinrutschen zu verhindern, kann man sie von unten mit einem langen Gegenstand fixieren. 
+  Die Silikonfuge kann man glätten, indem man einen Finger mit Spülmittel benetzt und damit das „Dach“ glättet. 
+
+.. Ab hier nicht mehr auskommentiert die Anleitung zur Auflage:
+
+Eine Zwischenbodenauflage dient zum Schutz vor Insekten, da sie das Loch der Kabeldurchführung verkleinert. Man kann dafür z.B. den roten Deckel der Kruke oder ein ähnliches Stück Kunststoff verwenden.
+
+Es wird die Mitte markiert und dort ein Loch mit dem Durchmesser des Stromkabels (ca. 5 mm) gebohrt. Anschließend werden die Ecken abgeschnitten. Zusätzlich wird ein Verbindungsschlitz von außen zum Loch geschnitten.
+
+.. |build15| image:: images/build_15_intermediateFloorSeal.jpg
+   :alt: Aufbau der Zwischenbodenauflage (in zwei Arbeitsschritten)
+
+|build15|
+Aufbau der Zwischenbodenauflage (in zwei Arbeitsschritten)
+
+Installationsplatte
+-------------------
+
+* Die Installationsplatte kann aus dem Ober- oder Unterteil eines Kabelkanals gefertigt werden. 
+  Dazu werden die Seitenteile vorsichtig mit einem Messer abgeschnitten.
+  Danach wird der dadurch entstandene „labbrige“ Kunststoffstreifen auf die richtige Länge gebracht. Hierzu kann eine starke Schere verwendet werden.
```

Comment:
labbrige -> flexible

Reply:
Ich habe das absichtlich nicht korrigiert, weil bei uns in Bayern würde man lapprig sagen und jedem sein Dialekt 😆

Reply:
naja, wir wollen ja auch nachbauer in anderen deutschsprachigen Gebieten das verständlich rüberbringen.

siehe auch "Kruke" (nie vorher gehört).